client/landing/stepper/declarative-flow/internals/videopress.scss

```diff
@@ -0,0 +1,237 @@
+@import "./global";
+
+$videopress-theme-base-text-color: #fff;
+$videopress-theme-background-color: #000;
+$videopress-theme-grey: #a7aaad;
+$videopress-theme-yellow: #ffe61c;
+$videopress-theme-header-subtitle-text-color: #a7aaad;
+$videopress-theme-border-color: #252726;
+
+body.is-videopress-stepper {
+	background-color: #010101;
+	color: $videopress-theme-base-text-color;
+
```

Comment:
```suggestion

	.search-filters__checkbox-label{
		color: #000;
	}
```

While looking at this flow I noticed that the checkbox in the filter is currently not visible.  This will make sure it's visible:
<img width="425" alt="Screenshot 2022-11-01 at 4 21 03 PM" src="https://user-images.githubusercontent.com/3220162/199360350-e3ea4441-ec01-4008-89cc-dd985c92856e.png">